elements/pfe-tabs/src/pfe-tab-panel.scss

```diff
@@ -92,3 +113,34 @@ $LOCAL-VARIABLES: (
 :host([hidden]) {
   display: none;
 }
+
+/// ===========================================================================
+/// CONTEXTS: DARK & SATURATED
+/// ===========================================================================
+/// In dark & saturated contexts, we override the local color vars
+
+
+:host {
+  @at-root #{&}([pfe-variant="earth"][on="dark"]) {
+    --pfe-tabs__panel--BackgroundColor: #{pfe-var(surface--darkest)};
```

Comment:
Should be setting theme or text color at the same time for this to work well.

Reply:
++ updated